mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerLocalScan.java

```diff
@@ -135,13 +135,13 @@ public void testScanTable() throws IOException {
     // Single fetch task: no MR job.
     List<Object[]> rows = shell.executeStatement("SELECT * FROM default.customers");
 
-    Assert.assertEquals(3, rows.size());
-    Assert.assertArrayEquals(new Object[] {0L, "Alice", "Brown"}, rows.get(0));
-    Assert.assertArrayEquals(new Object[] {1L, "Bob", "Green"}, rows.get(1));
-    Assert.assertArrayEquals(new Object[] {2L, "Trudy", "Pink"}, rows.get(2));
+    assertThat(rows).hasSize(3);
+    assertThat(rows.get(0)).isEqualTo(new Object[] {0L, "Alice", "Brown"});
```

Comment:
nit: this could also be written in a single stmt: `assertThat(rows).containsExactly(first, second, third)`